events/hfe_events_total_war.txt

```diff
@@ -139,7 +139,7 @@ planet_event = {
 # This = Ship
 # From = Disabler Ship
 # kill starbases of pirate and drone on_ship_disabled
-# проверить переключение на фром без рута
```

Comment:
In this file you just killed comments, should you revert changes

Reply:
Accidentally killing the comments was necessary. Multiple files, including this one, were corrupted. When I loaded them on my PC, an American Windows localization, it converted them to UTF and caused my game to crash repeatedly. That's why all the Cyrillic is replaced with question marks. My computer cannot render Cyrillic in a native fashion, so it has to switch to Unicode or UTF in order to depict it. I am sorry that those were removed and replaced with question marks, but the Mod was corrupted on my end because of the ANSI/UTF differences.